main/src/flix/runtime/library/MonoStringBuilder.java

```diff
@@ -0,0 +1,102 @@
+/*

```

Comment:
We usually use the word "mono" to refer to the process of monomorphization whereby a generic class is instead with its actual type arguments. The StringBuilder class is actually not generic, but it does use alot of the overloaded methods, so the naming is slightly misleading.

How do you feel about coming up with some general naming scheme for when we have to write the wrappers?

We could simply call them all `JStringBuilder` or `StringBuilderWrapper` or similar?

Reply:
The question is not too important, since at the some point we will hopefully have a better import mechanism :)